.github/workflows/image.yaml

```diff
@@ -73,12 +79,12 @@ jobs:
 
       - name: docker images publish
         run: |
-          make docker-publish-kyverno
+          make docker-publish-kyverno-dev
```

Comment:
@ShubhamPalriwala - can you help with my questions for image signature?
- `make docker-publish-sigs` is missing for kyverno and CLI images, do we need to add that?
- how is this used and do we need to sign the dev images as well? The "dev" image is used by users to verify fixes, features, etc.

Reply:
- So the major point of having `docker-publish-sigs` is to create a GHCR that can store the signatures, since the container is already created, we do not need it for kyverno and CLI
- Yeah, I'm assuming that we are pushing these to a separate container registry, let's sign these also so that these can also be verified, whenever needed by any user

Reply:
>we do not need it for kyverno and CLI

Ok.

>I'm assuming that we are pushing these to a separate container registry

No, it was added temporarily for testing purposes. I removed it from this PR.